IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -264,9 +267,9 @@ - (BOOL)saveAccount:(nonnull MSIDAccountCacheItem *)account
             return NO;
         }
     }
-
+    
     key.username = account.username;
```

Comment:
does this get set already in generateCacheKey?

Reply:
Updated code to not set user name in generateCacheKey for MSIDAccountCacheItem to reflect the previous logic.